www/src/pages-markdown/download.md

```diff
@@ -4,26 +4,26 @@ date: "2020-02-01"
 path: "/download"
 header: true
 ---
- - [Play Now (Firefox and Chrome)](/game)
 
-To run the game locally on your computer, use the following links:
+<div style="text-align: center;">
+    <img src="../assets/play-now.gif" width="50%"/>
```

Comment:
this is definitely going to look like trash on the phone

Reply:
![localhost_8000_(iPhone X) (4)](https://user-images.githubusercontent.com/2229300/86047142-47c54400-ba03-11ea-877f-df99d14250f6.png)

maybe also record portrait orientation gif and crop it so that we see a little bit better

![localhost_8000_(iPhone X) (5)](https://user-images.githubusercontent.com/2229300/86047263-80651d80-ba03-11ea-82a3-58797858377e.png)


on landscape looks decent, still lots of dead area, we'd expec tthis to be edge to edge. would be better if play now floated over the image with the clip also you can't play now on mobile using that link, so maybe use a css `media` trick to change that URL to be the testflight link on mobile

Reply:
also this is the old version of the play now gif
also it needs to be upscaled to be larger. you will probably want to do this using `convert` the imagemagick command line utility which can deal with gif nearest-neighbor scaling correctly

Reply:
it should not be fuzzy when it's scaled up

Reply:
maybe use a react component specific for this effect, which will typically require using a `.mp4` file